src/test/storage/segment_accessor_test.cpp

```diff
@@ -14,25 +14,28 @@ namespace opossum {
 class SegmentAccessorTest : public BaseTest {
  protected:
   void SetUp() override {
-    vc_int = make_shared_by_data_type<BaseValueSegment, ValueSegment>(DataType::Int, true);
-    vc_int->append(4);
-    vc_int->append(6);
-    vc_int->append(3);
-    vc_int->append(NULL_VALUE);
-
-    vc_str = make_shared_by_data_type<BaseValueSegment, ValueSegment>(DataType::String, true);
-    vc_str->append("Hello,");
-    vc_str->append("world");
-    vc_str->append("!");
-    vc_str->append(NULL_VALUE);
-
-    dc_int = ChunkEncoder::encode_segment(vc_int, DataType::Int, SegmentEncodingSpec{EncodingType::Dictionary});
-    dc_str = ChunkEncoder::encode_segment(vc_str, DataType::String, SegmentEncodingSpec{EncodingType::Dictionary});
-
-    tbl = std::make_shared<Table>(TableColumnDefinitions{TableColumnDefinition{"vc_int", DataType::Int, false},
-                                                         TableColumnDefinition{"dc_str", DataType::String, false}},
-                                  TableType::Data);
-    tbl->append_chunk({vc_int, dc_str});
+    val_seg_int = std::make_shared<ValueSegment<int>>(true);
```

Comment:
Not a big fan of this either, but requiring full names in tests maybe goes a bit too far. Just changed it because "vc" (for ValueColumn, the old name) was outdated.